### ISSUE TITLE
change to hosted dev environment

### ISSUE THREAD

**Issue by matthewberryman** [urbanriskmap/cognicity-analytics#28]
Now that field office and partners are moved to the migration I will work on updating the hostnames but you guys may want to change the auth0 domain and client_id over to the ones used for rem-dev (auth0 domain: petabencana-dev) so that the field office can also have logins for testing new features in dev. If necessary I can send you guys logins for that, although @talltom is an admin for that auth0 domain.

**Comment by matthewberryman** [comment]
Note to self to also update jenkins builds

**Comment by matthewberryman** [comment]
Done this in both master and dev branches (also noting merged other recent work in master into dev), and have now have dev branch being built using dev environment.ts -> dashboard-dev.petabencana.id
leaving master branch being built with environment.prod.ts -> dashboard.petabencana.id

**Comment by ojha-url** [comment]
Created a separate stage environment (environment.stage.ts) for dashboard-dev.petabencana.id
Updated config on jenkins.

Dev env, environment.ts to be used on local, bypasses auth lock as requested by @abrahamq 

I don't have access to rem-dev (auth0 domain: petabencana-dev), for now using the tenant + client + user I had created:
auth0_domain: 'dev-riskmap.auth0.com'
'Dashboard dev user' credentials shared on LastPass

@talltom, we can move this to rem-dev at some point. Closing issue for now.
